### DISCUSSION TITLE
Generic docker tag

### DISCUSSION THREAD

**Discussion by Norsagir** [itzg/docker-minecraft-server#2245] [Ideas]
Hi,
I was wondering if there was a way to update the docker image via WatchTower.
My problem is that the "latest" tag contains too much commits, so I select your latest release tag on Github and update manually.

So my question is : could you create a docker tag such as "2023" or "stable" that is the same image as the latest release using Java 17, Ubuntu, Hotspot for amd64,arm64,armv7 ?

That way, we would be able to update automatically our images with the last stable release, and not using the "latest" tag with many commits.

**Comment by itzg** [comment]
That's a great idea. I'll spin off an issue to track this.

**Comment by itzg** [comment]
A `stable` tag is now provided: https://hub.docker.com/repository/docker/itzg/minecraft-server/tags?page=1&ordering=last_updated&name=stable

**Reply by Norsagir** [reply]
Oh wow you are fast ! Thank you really much for this, my Minecraft server will now be up to date everyday ! :)

**Comment by Bobrokus** [comment]
I would rename `latest` to `nightly`. Makes more sense imo.

**Reply by itzg** [reply]
"latest" has special behavior in docker, so it's not arbitrary to rename. Over the 12+ years this project has been going "latest" tracks the latest commit on main branch, so that's also not something that can be casually changed.

**Reply by Bobrokus** [reply]
oh ok i see
